sceptre/cli/helpers.py

```diff
@@ -0,0 +1,267 @@
+    :type name: str
+    """
+    stack.create_change_set(name)
+    try:
+        yield
+    finally:
+        stack.delete_change_set(name)
+
+
+def get_stack_and_env(ctx, path, recursive):
+    """
+    Parses the path to generate relevant Envrionment and Stack object.
+
+    :param ctx: Cli context.
+    :type ctx: click.Context
+    :param path: Path to either stack config or environment folder.
+    :type path: str
+    :param recursive: Flag to allow environment level actions.
+    :type recursive: bool
+    """
+    stack_name = None
+
+    path, ext = os.path.splitext(path)
+    if ext:
+        path, stack_name = os.path.split(path)
+
+    env = Environment(ctx.obj["sceptre_dir"], path, ctx.obj["options"])
+
+    if stack_name:
+        try:
+            stack = env.stacks[stack_name]
+        except KeyError:
+            raise StackConfigNotFoundError()
+        return (stack, env)
+
+    elif not stack_name and not recursive:
+        raise RecursiveFlagMissingError("Use recursive flag.")
+
+    return (None, env)
+
+
+def get_stack(ctx, path):
+    """
+    Retrieves the Stack object for path to stack config.
+
+    :param stack: The stack to create the change set for.
+    :type stack: sceptre.stack.Stack
+    :param name: The name of the change set.
+    :type name: str
+    """
+    stack, _ = get_stack_and_env(ctx, path, False)
+    if not stack:
+        raise StackPathRequiredError("Path to a stack required.")
+    return stack
+
+
+def setup_logging(debug, no_colour):
+    """
+    Sets up logging.
+
+    By default, the python logging module is configured to push logs to stdout
+    as long as their level is at least INFO. The log format is set to
+    "[%(asctime)s] - %(name)s - %(message)s" and the date format is set to
+    "%Y-%m-%d %H:%M:%S".
+
+    After this function has run, modules should:
+
+    .. code:: python
+
+        import logging
+
+        logging.getLogger(__name__).info("my log message")
+
+    :param debug: A flag indication whether to turn on debug logging.
+    :type debug: bool
+    :no_colour: A flag to indicating whether to turn off coloured output.
+    :type no_colour: bool
+    :returns: A logger.
+    :rtype: logging.Logger
+    """
+    if debug:
+        sceptre_logging_level = logging.DEBUG
+        logging.getLogger("botocore").setLevel(logging.INFO)
+    else:
+        sceptre_logging_level = logging.INFO
+        # Silence botocore logs
+        logging.getLogger("botocore").setLevel(logging.CRITICAL)
+
+    formatter_class = logging.Formatter if no_colour else ColouredFormatter
+
+    formatter = formatter_class(
+        fmt="[%(asctime)s] - %(name)s - %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S"
+    )
+
+    log_handler = logging.StreamHandler()
+    log_handler.setFormatter(formatter)
+    logger = logging.getLogger("sceptre")
+    logger.addHandler(log_handler)
+    logger.setLevel(sceptre_logging_level)
+    return logger
+
+
+def simplify_change_set_description(response):
```

Comment:
Is there a good reason for defining this function here and describe.py?

Reply:
Yup duplicated function - removed from describe.py and imported from here